From 6581577d9092bb6bddba8c0442ad0e388d49205a Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Thu, 23 Nov 2017 00:21:13 +0000
Subject: [PATCH] CMake: Fix detection of Python 2

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8daac8a..29802df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ if (NOT PYTHON_DESIRED)
 endif()
 
 if (${PYTHON_DESIRED} STREQUAL "2")
-	FIND_PACKAGE (PythonInterp REQUIRED)
+	FIND_PACKAGE (PythonInterp 2.7 REQUIRED)
 else()
 	SET(Python_ADDITIONAL_VERSIONS 3.3)
 	FIND_PACKAGE(PythonLibs 3.0)
-- 
2.16.2

